umh-core/examples/example-config-empty.yaml

```
@@ -19,4 +19,4 @@ agent:
 internal:
   redpanda:
     desiredState: stopped
-  services: []
+  services: []
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)